tests/x11/yast2_lan_restart.pm

```diff
@@ -70,11 +70,66 @@ sub change_hw_device_name {
     send_key $cmd{next};
 }
 
+sub set_static_ip {
+    my $looprun = shift;
+    send_key 'alt-i';
+    assert_screen 'yast2_lan_network_card_setup';
+    send_key 'alt-t';
+    send_key 'tab';
+    type_string '192.168.1.10';
+    send_key 'tab';
+    type_string '255.255.255.0';
+    send_key 'tab';
+    type_string "test-$looprun.susetest.com";
+    wait_still_screen;    # yes, necessary.
+    assert_screen "gnome_static_IP_set";
+    send_key 'alt-n';
+}
+
+sub set_dhcp {
+    send_key 'alt-i';
+    send_key 'alt-y';
+    assert_screen 'yast2_lan_restart_dhcp_set';
+    send_key 'alt-n';
+}
+
+
+sub check_etc_hosts_update {
+
+=Description
```

Comment:
does this work as [perl POD](https://perldoc.perl.org/perlpod.html)? format looks strange

Reply:
I will change this to proper pod format.